### PR TITLE
Helena W

### PR DESCRIPTION
**Pull request by HelenaW86**
https://epic-leakey-4d6fbc.netlify.app/

### CODE REVIEW

**Comment by HelenaW86** [comment]
Link: https://epic-leakey-4d6fbc.netlify.app/

File: code/index.html [hunk 1]
```diff
@@ -3,28 +3,142 @@
...
+          </label>
+          <label for="unicorn">Unicorn
+            <input class="radio" type="radio" id="unicorn" name="what are you?" value="unicorn">
+          </label>
+        </div>
+        <div class="agrees">
+          <label class="checkbox">
+            I agree to the Terms & Conditions
+            <input class="ckeck" type="checkbox" name="pollicyagree" required>
+            <span class="checkmark"></span>
```
Comment (JuliaNiki) [inline code/index.html:86]: I am not sure you needed `<span>` here, but looks good anyway :)

File: code/script.js [hunk 2]
```diff
@@ -1,9 +1,21 @@
...
 }
 
 // Selects an HTML element, and calls a function which will be executed when the element is clicked.
-document.getElementById("section1").onclick = toggle
+
+
+document.getElementById('question').onclick = toggle
+document.getElementById('question1').onclick = toggle
+document.getElementById('question2').onclick = toggle
+document.getElementById('question3').onclick = toggle
```
Comment (JuliaNiki) [inline code/script.js:15]: Very clean and accurate Javascript code, nice job 👍🏼

File: code/style.css [hunk 3]
```diff
@@ -1,7 +1,339 @@
...
+  border: none;
+  border-radius: 5px;
+}
+
+select {
+  font-size: 20px;
+  margin: 5px;
+}
+
+input:focus {
```
Comment (JuliaNiki) [inline code/style.css:113]: Great that you've succeeded with Red level requirement and added focus on input tag

File: code/style.css [hunk 4]
```diff
@@ -1,7 +1,339 @@
...
+  0% {
+      background-position: 0% 50%;
+  }
+  50% {
+      background-position: 100% 50%;
+  }
+  100% {
+      background-position: 0% 50%;
+  }
+}
```
Comment (JuliaNiki) [inline code/style.css:180]: Oh wow! I have not noticed the animation on button at first, but cool!  My advise is to try to use some more visible colors next time or put animation on button without hovering so the user pays attention while doing the form :)

File: code/style.css [hunk 5]
```diff
@@ -1,7 +1,339 @@
...
+    width: 250px;
+  }
+
+  .logo {
+    width: 250px;
+    margin-top: 30vh;
+  }
+
+.form-and-faq-container {
+  flex-direction: row;
```
Comment (JuliaNiki) [inline code/style.css:279]: As a tip, try to follow the same amount of space from the left before you type classes, you can see that `.slogan` is a bit closer to the left than `.logo` :)

File: code/style.css [hunk 6]
```diff
@@ -1,7 +1,339 @@
...
+  }
+
+  .form {
+    padding-left: 70px;
+  }
+}
+
+
+/* font-family: 'Courgette', cursive;
+font-family: 'Ma Shan Zheng', cursive; */
```
Comment (JuliaNiki) [inline code/style.css:339]: If you do not need this code, try to delete it before you submit the final version :)

File: README.md [hunk 7]
```diff
@@ -1,12 +1,12 @@
...
-Replace this readme with your own information about your project. 
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+ 
+This weeks project was to build a bussines page whit a form and a accordion and a background picture or video, 
 
 ## The problem
 
 Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+If I had more time I would make the logo spin 360deg, That would be a cool effect I think. I had no big struggle, but the possition rellative and absolute does mather =D I have really learnd that now.
```
Comment (JuliaNiki) [inline README.md:9]: That would be sooo cool! 
You gave me inspiration to try it out also on some of the future projects :)

File: code/index.html [hunk 8]
```diff
@@ -3,28 +3,142 @@
...
+        <div class="contact-details">
+          <h3 class="form-title">contact details</h3>
+          <label for="name">Your Name:
+            <input type="text" name="name" required placeholder="Name">
+          </label>
+          <label for="email" > your Email:
+            <input type="email" name="email" required placeholder="Email">
+          </label>
+          <label for="Phonenumber">Your Phone number:
+            <input type="text" name="phonenumber" required placeholder="Phone number">
```
Comment (JuliaNiki) [inline code/index.html:50]: for the phone number you can use instead `<input type="number" />` so that the user cannot type any text :)

**Review by JuliaNiki — APPROVED** [review]
Very nice website, Helena! It makes me feel relaxed looking at it thanks to these waves haha :) 
The code looks clean and well organised. I only recommend to pay attentions to tabs so that the code looks even more readable.

**Pull request closed without merging**